ini-file-parser/src/test/java/com/baeldung/inifileparser/TestIniFileParser.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.inifileparser;
+
+import java.io.IOException;
+
+import org.ini4j.InvalidFileFormatException;
+import org.junit.Test;
+
+public class TestIniFileParser {
+
+    @Test
+    public void testIni4jParser() throws InvalidFileFormatException, IOException {
+        IniFileParser fileParser = new IniFileParser();
+        System.out.println(fileParser.parseIniFileUsingIni4j("C:/Baeldung/sample_config.ini"));
```

Comment:
This file doesn't exist on most people's machines. Put a file in `src/test/resources` and reference it from here.